modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,479 @@
+
+|--url
+|An RFC 2255 URL (string) which specifies the LDAP search parameters to use.
+
+|--username-attributes
+|The list (string) of attributes whose values should be used as the preferred username. Default: `uid`
+|===
+
+.OpenID arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--email-claims
+|The list (string) of claims to use as the email address.
+
+|--extra-scopes
+|The list (string) of scopes to request, in addition to the `openid` scope, during the authorization token request.
+
+|--issuer-url
+|The URL (string) that the OpenID provider asserts as the issuer identifier. It must use the HTTPS scheme with no URL query parameters or fragment.
+
+|--name-claims
+|The list (string) of claims to use as the display name.
+
+|--username-claims
+|The list (string) of claims to use as the preferred username when provisioning a user.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Examples
+Add a GitHub identity provider to a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa create idp --type=github --cluster=mycluster
+----
+
+Add an identity provider following interactive prompts.
+
+[source,terminal]
+----
+$ rosa create idp --cluster=mycluster --interactive
+----
+
+[id="rosa-create-ingress_{context}"]
+== create ingress
+
+Adds an ingress endpoint to enable API access to the cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create ingress --cluster=<cluster_name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to which the ingress will be added.
+
+|--label-match
+|The label match (string) for ingress. The format must be a comma-delimited list of `<key=value>` If no label is specified, all routes are exposed on both routers.
+
+|--private
+|Restricts application route to direct, private connectivity.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Examples
+Add an internal ingress to a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa create ingress --private --cluster=mycluster
+----
+
+Add a public ingress to a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa create ingress --cluster=mycluster
+----
+
+Add an ingress with a route selector label match.
+
+[source,terminal]
+----
+$ rosa create ingress -c mycluster --label-match="foo=bar,bar=baz"
```

Comment:
-c?

Reply:
A valid argument, but changed it to the long-form for consistency.